frappe/desk/form/meta.py

```diff
@@ -34,12 +34,14 @@
 )
 
 
-def get_meta(doctype, cached=True):
+def get_meta(doctype, cached=True) -> "FormMeta":
 	# don't cache for developer mode as js files, templates may be edited
-	if cached and not frappe.conf.developer_mode:
+	cached = cached and not frappe.conf.developer_mode
```

Comment:
This is fine, but my suggestion would be to use `frappe._dev_server` instead of `frappe.conf.developer_mode`. We should also probably remove duplicate check here:
https://github.com/frappe/frappe/blob/226cebe3b4d4cdbd27a77f26691f28c740fb824a/frappe/desk/form/load.py#L83